xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/Game.h

```diff
@@ -1462,6 +1462,16 @@ typedef struct KodiToAddonFuncTable_Game
   GAME_ERROR(__cdecl* CheatReset)(const AddonInstance_Game*);
   GAME_ERROR(__cdecl* GetMemory)(const AddonInstance_Game*, GAME_MEMORY, uint8_t**, size_t*);
   GAME_ERROR(__cdecl* SetCheat)(const AddonInstance_Game*, unsigned int, bool, const char*);
+  GAME_ERROR(__cdecl* RCGenerateHashFromFile)(const AddonInstance_Game*, char*, int, const char*);
```

Comment:
Should this take a size argument for the target string to guard against a buffer overflow?